cluster/manifests/admission-control/daemonset.yaml

```diff
@@ -0,0 +1,56 @@
+{{ if eq .Environment "production" }}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: admission-controller-proxy
+  namespace: kube-system
+  labels:
+    application: admission-controller-proxy
+spec:
+  selector:
+    matchLabels:
+      application: admission-controller-proxy
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        application: admission-controller-proxy
+    spec:
+      dnsConfig:
+        options:
+          - name: ndots
+            value: "1"
+      priorityClassName: system-cluster-critical
+      serviceAccountName: system
+      dnsPolicy: Default
+      hostNetwork: true
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        effect: NoSchedule
+      containers:
+      - name: cluster-autoscaler
+        image: registry.opensource.zalan.do/teapot/admission-controller:pr-14-9
+        command:
+          - /registry-proxy
+          - --address=127.0.0.1:8285
+          - --token-secret=/meta/credentials/admission-controller-token-secret
+          - --token-type=/meta/credentials/admission-controller-token-type
+        resources:
+          limits:
+            cpu: 25m
+            memory: 100Mi
+          requests:
+            cpu: 25m
+            memory: 100Mi
```

Comment:
probably `10-20Mi` is enough? This is ofc. on the master nodes, so not so bad, but I don't think we need `100Mi`

Reply:
I don't want to go lower than `50Mi` because we already had issues with a 3 line shell script, due to the cache memory being counted as well. Will update to 50, though!